extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -10,10 +10,10 @@ class PhalanxSpecialController extends WikiaSpecialPageController {
 	private $service;
 
 	public function __construct() {
-		parent::__construct('Phalanx', 'phalanx' /* restrictions */);
-		$this->includable(false);
+		parent::__construct( 'Phalanx', 'phalanx' /* restrictions */ );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Phalanx" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)